canal/handler.go

```diff
@@ -17,18 +18,22 @@ type EventHandler interface {
 	OnGTID(header *replication.EventHeader, gtid mysql.GTIDSet) error
 	// OnPosSynced Use your own way to sync position. When force is true, sync position immediately.
 	OnPosSynced(header *replication.EventHeader, pos mysql.Position, set mysql.GTIDSet, force bool) error
+	// OnQueryEvent is query event include(create user,drop user,create index event,etd.)
+	// Note: the OnQueryEvent has lower priority than OnDDL event
+	OnQueryEvent(ev *replication.BinlogEvent, e *replication.QueryEvent, stmt ast.StmtNode, pos *Position) error
```

Comment:
I prefer the parameters are (*EventHeader, StmtNode, *Position, *QueryEvent)

BinlogEvent contains QueryEvent, if we provide two ways to do a thing user may be confused.

Reply:
Well, okay